.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+language: java
+services:
+  - docker
+jdk:
+  - oraclejdk8
+os:
+  - linux
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+before_install:
+  - sudo apt-get install jq
+  - wget -O ~/codacy-coverage-reporter-assembly-latest.jar $(curl https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/latest | jq -r .assets[0].browser_download_url)
+  - docker build -t "draft-store:pgsql" ./docker/database/
+  - docker run -d -e DRAFT_STORE_DB_PASSWORD=$DRAFT_STORE_DB_PASSWORD -p $DRAFT_STORE_DB_PORT:5432 draft-store:pgsql
```

Comment:
is service2service auth not also needed for functional tests?

Reply:
seems to pass to im assuming not? :|

Reply:
@rkondratowicz should this be happening?